app/src/main/java/com/globant/counter/mvp/view/CalculatorView.kt

```diff
@@ -0,0 +1,98 @@
+package com.globant.counter.mvp.view
+
+import android.app.Activity
+import android.widget.Toast
+import com.globant.counter.rx.events.*
+import com.globant.counter.utils.Constants
+import com.globant.counter.utils.RxBus
+import kotlinx.android.synthetic.main.activity_main.*
+
+class CalculatorView(activity: Activity) : ActivityView(activity) {
+
+    init {
+        // Number click listeners
+        activity.btn_calculator_one.setOnClickListener {
```

Comment:
you could avoid creating soooo many click listeners if you just did something like
```suggestion
        val btnListener: (View) -> Unit = {
            RxBus.post(OnNumberButtonPressedBusObserver.OnNumberButtonPressed((View as TextView).text.toLong())
        }
        activity.btn_calculator_one.setOnClickListener(btnListener)
        ...
```